src/trilinos_interfaces/test_kokkos_init.f90

```diff
@@ -0,0 +1,19 @@
+!++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
```

Comment:
Shouldn't this and `test_trilinios_interfaces` really live in the `unit_tests` directory instead of the `src` directory?

Reply:
I would think so, but I just used the existing infrastructure. I can move both `test_trilinos_interfaces` and `test_kokkos_init` into the `unit_tests` directory if you think that's best.

Reply:
It makes sense to me, but I'm not sure if there was a particular reason they were placed here instead.  @stimpsonsg , @bscollin any comment on that?

Reply:
yeah, I'd make a directory in `Futility/unit_tests` to house these.  I don't really love that `trilinos_interfaces` is a separate directory within source, but not a huge deal, so long as it's consistent.

Reply:
Ok this change is pushed up.